app/helpers/invites_helper.rb

```diff
@@ -8,4 +8,8 @@ def invites_max_uses_options
   def invites_expires_options
     [30.minutes, 1.hour, 6.hours, 12.hours, 1.day, 1.week]
   end
+
+  def invite_never_expires
+    '∞'
```

Comment:
Options:

|Thing|Description|
|---|---|
|Infinity emoji|♾️|
|Infinity character|∞|

Reply:
Is more emoji useful here?